talisman/talisman.py

```diff
@@ -178,6 +195,7 @@ def _set_frame_options_headers(self, headers):
     def _set_content_security_policy_headers(self, headers):
         headers['X-XSS-Protection'] = '1; mode=block'
         headers['X-Content-Type-Options'] = 'nosniff'
+        headers['X-Download-Options'] = 'noopen'
```

Comment:
I added the header to the README, and will add a toggle. Do you prefer to make it disabled by default?